runtime/startup_byt.c

```diff
@@ -374,59 +374,53 @@ static void do_print_config(void)
   char_os * dir;
 
   /* Print the runtime configuration */
-  printf("version: %s\n"
-         "standard_library_default: %s\n"
-         "standard_library: %s\n"
-         "int_size: %d\n"
-         "word_size: %d\n"
-         "os_type: %s\n"
-         "host: %s\n"
-         "flat_float_array: %s\n"
-         "supports_afl: %s\n"
-         "windows_unicode: %s\n"
-         "supports_shared_libraries: %s\n"
-         "no_naked_pointers: %s\n"
-         "profinfo: %s\n"
-         "profinfo_width: %d\n"
-         "exec_magic_number: %s\n",
-           OCAML_VERSION_STRING,
-           caml_stat_strdup_of_os(OCAML_STDLIB_DIR),
-           caml_stat_strdup_of_os(caml_get_stdlib_location()),
-           8 * (int)sizeof(value),
-           8 * (int)sizeof(value) - 1,
-           OCAML_OS_TYPE,
-           HOST,
+  printf("version: %s\n", OCAML_VERSION_STRING);
+  printf("standard_library_default: %s\n",
+         caml_stat_strdup_of_os(OCAML_STDLIB_DIR));
+  printf("standard_library: %s\n",
+         caml_stat_strdup_of_os(caml_get_stdlib_location()));
+  printf("int_size: %d\n", 8 * (int)sizeof(value));
+  printf("word_size: %d\n", 8 * (int)sizeof(value) - 1);
+  printf("os_type: %s\n", OCAML_OS_TYPE);
+  printf("host: %s\n", HOST);
+  printf("flat_float_array: %s\n",
 #ifdef FLAT_FLOAT_ARRAY
-           "true",
+         "true");
 #else
-           "false",
+         "false");
```

Comment:
My editor gets annoyed at the extra `)` but I found that putting it below was ugly and having the `printf` in each branch made scanning from one to the next harder (not to mention that having the key values entered twice in the code is "risky")